### DISCUSSION TITLE
参数验证对Double类型无效

### DISCUSSION THREAD

**Discussion by hunao0221** [zuihou/zuihou-admin-cloud#304] [企业版] [answered]
lamp-web-pro的版本为4.14.0

<img width="1138" alt="image" src="https://github.com/dromara/lamp-cloud/assets/11972530/6823b331-e54d-4598-9707-cacc550f2d69">
<img width="925" alt="image" src="https://github.com/dromara/lamp-cloud/assets/11972530/69ca9761-3fe9-4615-8b0f-83ed843e40c6">

**Comment by zuihou** [comment, marked as answer]
<img width="803" alt="image" src="https://github.com/dromara/lamp-cloud/assets/8500410/5c153132-4ad4-4394-8992-e333edb9ffa3">

产生原因：
 formValidateService.ts  在处理规则时，字段类似是 Float、Double 等数字类型时，会自动加上这个校验规则。

当组件使用Input框时，表单校验组件获取的值是“字符串”类型（就算是小数），所以会校验失败。

解决方案2选1：
1. 注释 formValidateService.ts  中的上述代码
2. 使用 InputNumber 组件
<img width="526" alt="image" src="https://github.com/dromara/lamp-cloud/assets/8500410/dbbaf4b6-0328-4e62-a7ab-f99d27a16ddc">
